aws/resource_aws_cloudwatch_log_stream.go

```diff
@@ -60,7 +62,24 @@ func resourceAwsCloudWatchLogStreamRead(d *schema.ResourceData, meta interface{}
 
 	group := d.Get("log_group_name").(string)
 
-	ls, exists, err := lookupCloudWatchLogStream(conn, d.Id(), group, nil)
+	var ls *cloudwatchlogs.LogStream
+	var exists bool
+
+	err := resource.Retry(2*time.Minute, func() *resource.RetryError {
```

Comment:
I'm not really sure what a good upper bound for the retry timeout would be. In automation today, we do a second Terraform apply after the first one fails and that has always succeeded - probably well under 2 minutes.

Reply:
Two minutes is pretty reasonable 👍 We can/will likely add a constant for this in the future.